tests/factories.py

```diff
@@ -18,10 +18,10 @@ class Meta:  # pylint: disable=too-few-public-methods
 
     id = factory.Sequence(lambda n: n)
     name = factory.Faker("first_name")
-    user_id = FuzzyChoice(choices=["1", "2", "3", "4"])
-    product_id = FuzzyChoice(choices=["1", "2", "3", "4"])
-    quantity = FuzzyChoice(choices=[1.0, 2.0, 3.0, 4.0])
-    price = FuzzyChoice(choices=[1.0, 2.0, 3.0, 4.0])
+    user_id = str(FuzzyInteger(1, 999))
+    product_id = str(FuzzyInteger(1, 999))
+    quantity = FuzzyInteger(1, 999)
```

Comment:
quantity here should be float, since it's float in the database schema

Reply:
Quantity must be an integral number.
I don't know how to generate integral float in FuzzyFloat?

Reply:
Then maybe we should use the old method to generate random quantity. Since it makes more sense to make quantity float, and it's float in database.

Reply:
Or I have an idea. Is it possible to convert the FuzzyInteger to float and add 0.5 to it before convert it to string.